UI/AppKit/Application/ApplicationDelegate.mm

```diff
@@ -628,6 +628,9 @@ - (NSMenuItem*)createDebugMenu
     auto* menu = [[NSMenuItem alloc] init];
     auto* submenu = [[NSMenu alloc] initWithTitle:@"Debug"];
 
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Dump Display List"
+                                                action:@selector(dumpDisplayList:)
+                                         keyEquivalent:@""]];
```

Comment:
The order of the dump operations was from more abstract to more concrete, i.e. DOM > Layout > Paint etc., I think Dump Display List should go last instead of first here.